test/elfeed-protocol-ttrss-test.el

```diff
@@ -105,3 +108,20 @@
                      (elfeed-entry-tags entry1)
                      '(star tag1 unread)))
             ))))))
+
+(ert-deftest elfeed-protocol-ttrss-parse-categories ()
```

Comment:
This test code is good. And feel free to fill the more complete test in `elfeed-protocol-fever-parse-entries` :)

Reply:
You mean `elfeed-protocol-ttrss-parse-entries`? Changed the feeds.json and added the tags here.